pug/test/terms.pug

```diff
@@ -2,7 +2,7 @@
 	md-toolbar
 		.md-toolbar-tools(layout="row", layout-align="space-between center")
 			h2(flex="20") Terms & Conditions
-			img(src="/images/3drepo-logo-white.png")
+			img(src="/images/3drepo-logo-white.png" alt="3drepo-logo-white")
```

Comment:
This alt text has hyphens in place of spaces and has no capitalisation.

Reply:
Me and Jozef where attempting to the get the checker to pass by providing what they call a "descriptive alt text". I am going to change it back to how the other are as it didn't seem to make a difference.